src/server/api/schema.js

```diff
@@ -1060,8 +1060,10 @@ const rootMutations = {
     updateQuestionResponses: async (
       _,
       { questionResponses, campaignContactId },
-      { loaders }
+      { loaders, user }
     ) => {
+      // TODO: check that the user has TEXTER role for the campaign
+      await authRequired(user);
```

Comment:
`authRequired` isn't async.  I suppose we need to do a assignmentRequiredOrAdmin lookup.

Reply:
updated